padertorch/configurable.py

```diff
@@ -523,18 +523,26 @@ def fix_doctext_import_class(locals_dict):
         >>> import_class(class_to_str(Foo))  # doctest: +ELLIPSIS
         Traceback (most recent call last):
         ...
-        AttributeError: Module ...configurable... has no attribute Foo. ...
+        ImportError: Could not import 'Foo' from '...configurable',
+        because module '...configurable' has no attribute 'Foo'
+        <BLANKLINE>
+        Make sure that
+         1. This is the class you want to import.
+         2. You activated the right environment.
+         3. The module exists and has been installed with pip.
+         4. You can import the module (and class) in ipython.
+        <BLANKLINE>
         >>> fix_doctext_import_class(locals())
-        >>> class_to_str(Foo)
-        'padertorch.configurable.Foo'
-        >>> class_to_str(foo)
-        'padertorch.configurable.foo'
-        >>> Foo
-        <class 'padertorch.configurable.Foo'>
+        >>> class_to_str(Foo)  # doctest: +ELLIPSIS
+        '...configurable.Foo'
+        >>> class_to_str(foo)  # doctest: +ELLIPSIS
+        '...configurable.foo'
+        >>> Foo  # doctest: +ELLIPSIS
+        <class '...configurable.Foo'>
         >>> import_class(class_to_str(foo)) == foo
         True
-        >>> import_class(class_to_str(Foo))
-        <class 'padertorch.configurable.Foo'>
+        >>> import_class(class_to_str(Foo))  # doctest: +ELLIPSIS
+        <class '...configurable.Foo'>
     """
```

Comment:
why do you need elipsis here? Shouldnt Configurable always be imported from padertorch?

Reply:
This is a bug in pycharm. In pycharm it is `configurable.Foo`, while pytest has the correct `padertorch.configurable.Foo`.

The bug is reported, but they don't care:
https://youtrack.jetbrains.com/issue/PY-26738